Helper/DeletedProductSerializer.php

```diff
@@ -15,7 +15,7 @@ public function serialize($deletedProductOrders)
                 $parentItemId   = $item->getParentItemId();
                 $itemId         = $item->getProductId();
                 $itemSku        = $item->getSku();
-                $itemName       = $item->getname();
```

Comment:
Wow. It was like that before?

Reply:
yes

Reply:
Jesus... it is a bug, right?

Reply:
Well long story short - PHP. It was a bug from semantic point of view but In PHP, Variable names are case-sensitive but function names are case-insensitive, though it is usually good form to call functions in the same case as they appear in their declaration.

Reply:
🤦‍♂